less/about/preferences.less

```diff
@@ -14,7 +14,9 @@ body {
 }
 
 #paymentsSwitches {
-  margin-top: 15px;
+  display: flex;
+  align-items: center;
+  min-height: 40px;
```

Comment:
Fixed into:
<img width="773" alt="screenshot 2016-12-01 17 06 54" src="https://cloud.githubusercontent.com/assets/3362943/20786316/9822529c-b7e8-11e6-9f61-dd65357adb7d.png">

The label of the button "off" is slightly up. Will fix.